recipes/ampliconsuite/meta.yaml

```diff
@@ -20,28 +20,26 @@ source:
 
 build:
     noarch: python
-    number: 0
+    number: 1
 
 requirements:
     build:
         - python >=3
```

Comment:
those should be in the host section not in the build section.

Reply:
Hi, thanks for doing this review. Can you please show me some documentation that says the build number should be placed into the host section? The documentation I see from conda shows an example where the build number is nested under the build section:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build-number-and-string

Reply:
This here is not about the build number. But the dependency on python.